doc/source/cluster/kubernetes/examples/ml-example.md

```diff
@@ -14,11 +14,36 @@ workload on Kubernetes infrastructure.
 We will run Ray's {ref}`XGBoost training benchmark <xgboost-benchmark>` with a 100 gigabyte training set.
 To learn more about using Ray's XGBoostTrainer, check out {ref}`the XGBoostTrainer documentation <train-gbdt-guide>`.
 
-## Kubernetes infrastructure setup
+## Kubernetes infrastructure setup on GCP
+
+This document provides instructions for GCP to create a Kubernetes cluster, but a similar setup would work for any major cloud provider. If you have an existing Kubernetes cluster, you can ignore this step.
+
+```shell
+# Set up a cluster on Google Kubernetes Engine (GKE)
+gcloud container clusters create autoscaler-ray-cluster \
+    --num-nodes=10 --zone=us-central1-c --machine-type e2-standard-16 --disk-size 1000GB
+
+# (Optional) Set up a cluster with autopilot on Google Kubernetes Engine (GKE).
+gcloud container clusters create autoscaler-ray-cluster \
+    --num-nodes=1 --min-nodes 1 --max-nodes 10 --enable-autoscaling \
+    --zone=us-central1-c --machine-type e2-standard-16 --disk-size 1000GB
+```
+
+Make sure you are connected to your Kubernetes cluster. For GCP, you can do so by:
+* Navigate to your GKE cluster page, and click "CONNECT" button. Then, copy "Command-line access".
+* `gcloud container clusters get-credentials <your-cluster-name> --region <your-region> --project <your-project>` ([Link](https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials))
+* `kubectl config use-context` ([Link](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/))
+
+```{admonition} Optional: Set up an autopilot GKE cluster
+**If you would like to try running the workload with autoscaling enabled**, use an autoscaling
+node group or pool with a 1 node minimum and a 10 node maximum.
```

Comment:
Is this the same as `--num-nodes=1 --min-nodes 1 --max-nodes 10 --enable-autoscaling` as written above? Writing a more specific command or isntruction here would be ideal.

Reply:
Yes, I moved this comment to the `gcloud` command.

https://github.com/ray-project/ray/pull/33339/commits/362b7983edc37a7255bbb50154d2cb72d2068db3